Update Known-issues.md (#3080)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/Known-issues.md

|:-----|:-----|:-----|:-----|
|Audit logs may report an incorrect username as initiator when someone has been removed from a Team  <br/> |Teams team is a modern group in AAD. When you add/remove a member through the Teams user interface, the flow knows exactly which user initiated the change, and the Audit log reflects the correct info. However, if a user adds/removes a member through AAD, the change is synced to the Teams backend without telling Teams who initiated the action. Microsoft Teams picks the first owner of team as the initiator, which is eventually reflected in the Audit log as well.    <br/> |  <br/> |5/11/18  <br/> |

|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
|:-----|:-----|:-----|:-----|

> |Teams voicemail emails will arrive with spf fail if it is a sip call, if it is a pstn call to a user they will arrive with the from attribute with out the correct value, if the customer has a rule where he analizes the spf  voice mails will have the action where the etr decides. <br/> | <br/> | 29/08/2019 workaround will be addig a exception in the etr if the message is a voice mail.

Reviewer: ```suggestion
|Teams voicemail emails report an "SPF Fail" error if they are SIP calls. PSTN calls report an incorrect value for the"From" attribute. If the customer has a rule to analyze the SPF, The ETR will decide on the action for voicemails. <br/> | <br/> | 29/08/2019 workaround will be adding an exception in the ETR if the message is a voice mail.
```

Reviewer 2: added to this commit, per PM Marc Lauinger.
